hardware/wireless-chipsets/en.md

```diff
@@ -10,14 +10,14 @@ This list should not suggest that *only*- such devices listed below are compatib
 
 ## Atheros
 
-- Atheros AR242x / AR542x
 - Atheros AR5B97
+- Atheros AR242x / AR542x
```

Comment:
While we're sorting this out, let's go ahead and just put this on a newline. Not sure why it's being bundled with AR242x

Reply:
Seems like a good idea, fixed now :+1: